packages/modal/src/Modal.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+import Modal from './Modal';
+
+afterAll(cleanup);
+
+describe('packages/Modal', () => {
+  const modalId = 'modal';
+  const modalContent = 'Modal Content';
+
+  describe('when rendered with content', () => {
+    const { getByText, getByTestId } = render(
+      <Modal active data-testid={modalId}>
+        <h4>{modalContent}</h4>
+      </Modal>,
+    );
+
+    test('it appears on DOM when active prop is set', () => {
+      const modal = getByTestId(modalId);
+      expect(modal).toBeVisible();
```

Comment:
I don't think this is accurate to the test name – if this tests visibility, that's different than testing if it's mounted.

Reply:
that's why i used the word 'appears' i'm testing that it's visible in the DOM

Reply:
My confusion comes down to the test name I think. Things are `rendered` in the DOM, and `visible` on the page. This test doesn't test specifics about the DOM.

We should probably test both whether it's rendered, and then its visibility.